internal/auth/auth.go

```diff
@@ -133,18 +133,6 @@ func New() *Authenticator {
 	}
 }
 
-// Start kicks-off the device authentication flow by requesting
-// a device code from Auth0. The returned state contains the
-// URI for the next step of the flow.
-func (a *Authenticator) Start(ctx context.Context) (State, error) {
```

Comment:
A bit of a refactor here. Removing the `Start` function because it only exists as a function wrapper for `getDeviceCode`. Further, the name wasn't descriptive of what the function did.